app/src/main/java/emu/skyline/settings/SettingsActivity.kt

```diff
@@ -23,6 +29,7 @@ import emu.skyline.utils.WindowInsetsHelper
 
 class SettingsActivity : AppCompatActivity() {
     val binding by lazy { SettingsActivityBinding.inflate(layoutInflater) }
+    var mapKeyTitle = mutableMapOf<String, Preference>()
```

Comment:
You usually want variables to have meaningful names, in this case `titleToPreferenceMap`. Add a KDoc comment on top to explain why this exists.